scripts/default_config.py

```diff
@@ -618,6 +616,7 @@ def model_kwargs(cfg, num_classes):
         'type': cfg.model.type,
         'self_challenging_cfg': cfg.model.self_challenging_cfg,
         'hidden_dim': cfg.model.transformer.hidden_dim,
+        'similarity_adjustment': cfg.loss.am_binary.amb_t,
```

Comment:
It looks like similarity_adjustment should be boolean

Reply:
cfg.loss.am_binary.amb_t != 0 or smth as soon as you excluded cfg.loss.am_binary.sym_adjustment from config